packages/data-mate/src/function-configs/geo/geoPointWithinRange.ts

```diff
@@ -0,0 +1,74 @@
+import {
+    geoPointWithinRangeFP, isGeoPoint, joinList,
+    parseGeoDistance
+} from '@terascope/utils';
+import { FieldType, GeoPointInput, GEO_DISTANCE_UNITS } from '@terascope/types';
+import {
+    FieldValidateConfig,
+    ProcessMode,
+    FunctionDefinitionType,
+    FunctionDefinitionCategory
+} from '../interfaces';
+
+export interface GeoPointWithinRangeArgs {
+    point: GeoPointInput;
+    distance: string
+}
+
+export const geoPointWithinRangeConfig: FieldValidateConfig<GeoPointWithinRangeArgs> = {
+    name: 'geoPointWithinRange',
+    aliases: ['geoDistance'],
+    type: FunctionDefinitionType.FIELD_VALIDATION,
+    process_mode: ProcessMode.INDIVIDUAL_VALUES,
+    category: FunctionDefinitionCategory.GEO,
+    examples: [
+        {
+            args: { point: '33.435518,-111.873616', distance: '5000m' },
+            config: { version: 1, fields: { testField: { type: FieldType.String } } },
+            field: 'testField',
+            input: '33.435967,-111.867710',
+            output: '33.435967,-111.867710',
+        },
+        {
+            args: { point: '33.435518,-111.873616', distance: '5000m' },
+            config: { version: 1, fields: { testField: { type: FieldType.String } } },
+            field: 'testField',
+            input: '22.435967,-150.867710',
+            output: null,
+        },
+    ],
+    description: 'Checks to see if input geo-point is within range of the point and distance values provided',
+    create({ point, distance }) {
+        const fn = geoPointWithinRangeFP(point, distance);
```

Comment:
This is minor but doing it like you have loses the name of the function which is useful for debugging
```suggestion
        return geoPointWithinRangeFP(point, distance) as (input: unknown) => boolean;
```